internal/cmd/schema-registry/command_schema_delete.go

```diff
@@ -78,11 +78,13 @@ func deleteSchema(cmd *cobra.Command, srClient *srsdk.APIClient, ctx context.Con
 		checkVersion = "latest"
 	}
 	if permanent {
-		opts := &srsdk.GetSchemaByVersionOpts{Deleted: optional.NewBool(true)}
-		if _, httpResp, err := srClient.DefaultApi.GetSchemaByVersion(ctx, subject, checkVersion, opts); err != nil {
-			return errors.CatchSchemaNotFoundError(err, httpResp)
-		} else if _, _, err := srClient.DefaultApi.GetSchemaByVersion(ctx, subject, checkVersion, nil); err == nil {
-			return errors.New("you must first soft delete a schema version before you can permanently delete it")
+		if checkVersion != "latest" {
```

Comment:
What happens when both `--permanent` and `--version latest` are passed?

Reply:
`permanent` works on already soft-deleted schemas, and because of the bug, the call will return error for `latest` that's why we're bypassing it

Reply:
Actually, it looks like `--permanent` and `--version latest` is an invalid combination for the delete endpoint itself.

I'll check w/ the backend team to see if this is another bug like the one @MuweiHe mentioned for the get endpoint, or if this is expected behavior.

Reply:
oh i was trying to say already soft-deleted so `opts: deleted=true`. we on the same page?

Reply:
Yup. I was just noticing that this applies to the delete endpoint as well.